src/Symfony/Bundle/WebProfilerBundle/Controller/ExceptionController.php

```diff
@@ -99,10 +99,11 @@ public function cssAction($token)
 
         $this->profiler->disable();
 
+        $exception = $this->profiler->loadProfile($token)->getCollector('exception')->getException();
         $template = $this->getTemplate();
 
         if (!$this->templateExists($template, false)) {
-            return new Response($this->errorRenderer->getStylesheet(), 200, ['Content-Type' => 'text/css']);
+            return new Response($this->errorRenderer->getStylesheet($exception), 200, ['Content-Type' => 'text/css']);
         }
 
         return new Response($this->twig->render('@WebProfiler/Collector/exception.css.twig'), 200, ['Content-Type' => 'text/css']);
```

Comment:
The point of the ExceptionController inside the WebProfilerBundle is to render the exception stack trace/css in the profiler exception panel isn't it? But that would only make sense in the debug mode. It should not use the custom error twig pages in prod mode. So to me it looks like as if the whole twig logic inside this controller is useless. It would be enough to only use the ErrorRenderer.

Reply:
So we could also deprecate this controller, create a new one which only uses ErrorRenderer and change the profiler routing to use the new controller.

Reply:
But I would be ok to do that in a separate PR.

Reply:
Yes, let's do that in another PR, thanks.

Reply:
See https://github.com/symfony/symfony/pull/32695